recipes/abacas/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set name = "ABACAS" %}
+{% set version = "1.3.1" %}
+{% set sha256 = "0afee209a4f879987b320e2f882bc0eab540a7a712a81a5f01b9795d2749310d" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://sourceforge.net/projects/{{ name|lower }}/files/{{ name|lower }}.{{ version }}.pl
+  sha256: {{ sha256 }}
+  patches:
+    - patches/path.patch
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  host:
+    - perl # [not osx]
+  run:
+    - perl # [not osx]
+    - mummer
+
+test:
+  commands:
+    - abacas.pl 2>&1 | grep "abacas.pl" > /dev/null
+
+about:
+  home: http://abacas.sourceforge.net/
+  license: GPL-2
```

Comment:
GPL-2.0-only ?

Reply:
In the script it says:
`# This program is free software; you can redistribute it and/or
# modify it under the terms of the GNU General Public License
# as published by the Free Software Foundation; either version 2
# of the License, or (at your option) any later version.`

How do you think it's better to indicate it?